packages/easy-coding-standard/src/Config/ECSConfig.php

```diff
@@ -108,27 +120,104 @@ public function ruleWithConfiguration(string $checkerClass, array $configuration
                 $service->property($propertyName, $value);
             }
         }
+
+        return $this;
     }
 
     /**
      * @param Option::INDENTATION_* $indentation
      */
-    public function indentation(string $indentation): void
+    public function indentation(string $indentation): self
     {
         $parameters = $this->parameters();
         $parameters->set(Option::INDENTATION, $indentation);
+
+        return $this;
     }
 
-    public function lineEnding(string $lineEnding): void
+    public function lineEnding(string $lineEnding): self
     {
         $parameters = $this->parameters();
         $parameters->set(Option::LINE_ENDING, $lineEnding);
+
+        return $this;
+    }
+
+    public function cacheDirectory(string $cacheDirectory): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::CACHE_DIRECTORY, $cacheDirectory);
+
+        return $this;
+    }
+
+    public function cacheNamespace(string $cacheNamespace): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::CACHE_NAMESPACE, $cacheNamespace);
+
+        return $this;
+    }
+
+    /**
+     * @param string[] $fileExtensions
+     */
+    public function fileExtensions(array $fileExtensions): self
+    {
+        Assert::allString($fileExtensions);
+
+        $parameters = $this->parameters();
+        $parameters->set(Option::FILE_EXTENSIONS, $fileExtensions);
+
+        return $this;
     }
 
-    public function parallel(): void
+    public function parallel(): self
     {
         $parameters = $this->parameters();
         $parameters->set(Option::PARALLEL, true);
+
+        return $this;
+    }
+
+    public function parallelJobSize(int $parallelJobSize): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::PARALLEL_JOB_SIZE, $parallelJobSize);
+
+        return $this;
+    }
+
+    public function parallelPort(string $parallelPort): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::PARALLEL_PORT, $parallelPort);
+
+        return $this;
+    }
+
+    public function parallelIdentifier(string $parallelIdentifier): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::PARALLEL_IDENTIFIER, $parallelIdentifier);
+
+        return $this;
+    }
+
+    public function parallelMaxNumberOfProcesses(int $parallelMaxNumberOfProcesses): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::PARALLEL_MAX_NUMBER_OF_PROCESSES, $parallelMaxNumberOfProcesses);
+
+        return $this;
+    }
+
+    public function parallelTimeoutInSeconds(int $parallelTimeoutInSeconds): self
+    {
+        $parameters = $this->parameters();
+        $parameters->set(Option::PARALLEL_TIMEOUT_IN_SECONDS, $parallelTimeoutInSeconds);
+
+        return $this;
     }
```

Comment:
Could you move these as `parallel()` arguments?
The similar way we have them here: https://github.com/rectorphp/rector-src/blob/3168114f48cdce16b777985280185eb88f0ed3b6/packages/Config/RectorConfig.php#L53

So the related content is together in single method.

Reply:
Cool. Done.